MVP Module/Tests/Feed UI/FeedUISnapshotTests.swift

```diff
@@ -49,8 +49,8 @@ class FeedUISnapshotTests: XCTestCase {
 	}
 }
 
-private extension FeedViewController {
-	func display(errorMessage: String) {
-		fatalError("Must be implemented - follow the MVC solution as a guide")
-	}
-}
+//private extension FeedViewController {
+//	func display(errorMessage: String) {
+//		display(errorMessage)
+//	}
+//}
```

Comment:
Instead of commenting out this helper extension (and changing the test structure), you can use it as a place to create the `ViewModel`: so if on the future you decide to refactor from MVVM to another pattern you won't break your tests.
```suggestion
private extension FeedViewController {
	func display(errorMessage: String) {
		display(FeedErrorViewModel(errorMessage: errorMessage))
	}
}
```

Reply:
done